Marlin/pins_ULTIMAIN_2.h

```diff
@@ -110,3 +118,18 @@
 #define BTN_EN1            40
 #define BTN_EN2            41
 #define BTN_ENC            19
+
+// Rev B additions
+
+//
+// M3, M4 & M5 (spindle control) support
+//
+#if ENABLED(SPINDLE_LASER_ENABLE)   // use the LED_PIN for spindle speed control or case light
+  #undef  LED_PIN
+  #define SPINDLE_DIR_PIN    16
+  #define SPINDLE_LASER_ENABLE_PIN 17  // should have a pull up
+  #define SPINDLE_SPEED_LASER_POWER_PIN   8  // must have a pull up
+#else                        // Case light
```

Comment:
Yes, let's just set the most sensible default pins, remove the #if clauses referring to the spindle-laser feature, and see what we have left in the pins files after that. I used to frown on having pin assignments in the main configuration files, which is why we started making the pins files more "exhaustive" on their own.

We still haven't settled on how best to handle pin overrides. One universal method would be to have every pin definition be conditional on `#ifndef`, but that would add a lot of noise to the pins files. The extruder auto fan pins (`E0_AUTO_FAN_PIN`, etc.) follow a pattern that gives more control from the configs (since these pins also act as enable switches), but it's not the kind of technique you'd want to use for all pins.